kuma/users/providers/github/provider.py

```diff
@@ -29,17 +29,17 @@ class KumaGitHubProvider(GitHubProvider):
     account_class = KumaGitHubAccount
 
     def extract_email_addresses(self, data):
-        email_addresses = []
-        for email_address in data.get('email_addresses', []):
-            # let's ignore all email address that have not been verified at
-            # GitHub's side
-            if (not email_address.get('verified', False) or
-                    not email_address.get('email', '').strip()):
-                continue
-            email_addresses.append(EmailAddress(email=email_address['email'],
-                                                verified=True,
-                                                primary=email_address['primary']))
-        return email_addresses
+        result = []
```

Comment:
This rewrite wasn't necessary, but I hated the old code so much that I decided to clean it up.